.gitignore

```diff
@@ -42,3 +42,5 @@ jspm_packages
 # Build files
 /Chart.Financial.js
 /Chart.Financial.min.js
+/.vs/chartjs-chart-financial/v15/.suo
```

Comment:
Can you put these under the Visual Studio section above?
Also, I tried to ignore them all by ignoring the whole .vs directory. Did that not work? I'm surprised these are necessary. Could we fix that to ignore the whole directory somehow?